helm/charts/kratos/templates/statefulset-mail.yaml

```diff
@@ -54,7 +54,14 @@ spec:
         - name: {{ include "kratos.fullname" . }}-courier
           image: {{ include "kratos.image" . }}
           imagePullPolicy: {{ include "kratos.imagePullPolicy" . }}
-          args: [ "courier", "watch", "--config", "/etc/config/kratos.yaml" ]
+          args:
+            - courier
+            - watch 
+            - --config
+            - /etc/config/kratos.yaml
+            {{- if .Values.statefulset.extraArgs }}
```

Comment:
Please add the new parameter to the values.yaml with a default value of `[]`.

A good idea would be also to add some testing parameters to the values in the `/hack` directory, to ensure that the manifests are properly generated :)